tests/acceptance/features/useridentity-fraud-check-expiry.feature

```diff
@@ -0,0 +1,55 @@
+Feature: UserIdentity Fraud Check Expiry
+
+  Scenario: Record is not expired when fraud check is within validity period
+    Given the user has a fraud check credential with nbf 5 months ago
+    And the user has a stored identity, with VOT "P2"
+    When I make a request for the users identity with a VTR "P2"
+    Then the status code should be 200
```

Comment:
Let me know if you'd prefer `Then the status code should be 200 / And the stored identity should be returned` is removed from this feature file, as one could argue it's being tested elsewhere, and is not the point of this test.